src/Raven.Server/Monitoring/Snmp/Objects/Database/5.2.X/1/DatabaseCountOfDocuments.cs

```diff
@@ -0,0 +1,23 @@
+using Lextm.SharpSnmpLib;
+using Raven.Server.Documents;
+
+namespace Raven.Server.Monitoring.Snmp.Objects.Database
+{
+    public class DatabaseCountOfDocuments : DatabaseScalarObjectBase<Gauge32>
+    {
+        public DatabaseCountOfDocuments(string databaseName, DatabasesLandlord landlord, int index)
+            : base(databaseName, landlord, "5.2.{0}.1.4", index)
+        {
+        }
+
+        protected override Gauge32 GetData(DocumentDatabase database)
+        {
+            return new Gauge32(GetCount(database));
+        }
+
+        private static long GetCount(DocumentDatabase database)
+        {
+            return database.DocumentsStorage.GetNumberOfDocuments();
```

Comment:
Cheap, but each of these will open up a separate transaction, and if this is called a lot, may be worth optimizing.